modules/plugins/browser/index.js

```diff
@@ -40,6 +40,7 @@ function browserPluginFactory(opts = {}, browser = safeBrowser) {
 
 
         function updateBrowserState(state, url, replace) {
+            state = { ...browser.getState(), ...state };
```

Comment:
`state` is shared with other consumers. Can you do `const finalState = ...`? Thanks!